docs/Roadmap.md

```diff
@@ -0,0 +1,150 @@
+# ONNX Runtime Roadmap
+ONNX Runtime is an active, fast-paced project backed by a strong team of deeply technical Microsoft engineers along with a worldwide 
+community of partners and contributors. This roadmap summarizes the pending investments identified by the team to continually grow 
+ONNX Runtime as a robust, versatile, and high performance engine for DNN and traditional ML models.
+
+## High Level Goals
+ONNX Runtime fully aligns with the goals of the [ONNX](https://onnx.ai/) community to bring openness and interoperability to the AI industry. 
```

Comment:
Also consider if we should add some of the high level goals from this [slide 35](https://microsoft.sharepoint.com/:p:/r/teams/ONNXDiscussions/_layouts/15/Doc.aspx?sourcedoc=%7B6348F0A6-BA59-464D-A52C-53C737E2B029%7D&file=Aug2019%20-%20ORT%20overview%20for%20partners.pptx&action=edit&mobileredirect=true).

Reply:
I added a link to the technical design page, which covers some of those topics